open-machine-learning-jupyter-book/llm/basic/language-modelling.ipynb

```diff
@@ -0,0 +1,1724 @@
+{
```

Comment:
<p>Can this model be downloaded locally and uploaded to cos? Try to do so if it is possible.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/746/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Models can be downloaded locally and uploaded to cos.
But it is not possible to modify the code to load from cos directly. Here the transformers package is used to load the model, it is not possible to get the model from the cos link, only from local or huggingface.

only one way: download from hub and load from loacl. I don't think that's necessary.
import requests
from transformers import BertModel

model_url = "https://static-1300131294.cos.ap-shanghai.myqcloud.com/images/llm/bert.safetensors"
model_file = "bert_model.pth"
response = requests.get(model_url)
with open(model_file, "wb") as f:
    f.write(response.content)

BERTmodel = BertModel.from_pretrained(model_file)

Reply:
OK，good job！